Makefile

```diff
@@ -108,6 +108,48 @@ clean:
         clientapi/webrca \
 		$(NULL)
 
+MODULES = model clientapi metamodel_generator 
+.PHONY: validate-version
+validate-version:
```

Comment:
This target is written with dash ("-") while the others are using lower dash '_'. Can we align for consistency? It seems convention is dash.